examples/basic.ts

```diff
@@ -3,23 +3,23 @@ import erdosRenyi from 'graphology-generators/random/erdos-renyi';
 import randomLayout from 'graphology-layout/random';
 import chroma from 'chroma-js';
 import faker from 'faker';
-import WebGLRenderer from '../src/renderers/webgl';
+import WebGLRenderer from '../src/renderers/webgl/index';
```

Comment:
TypeScript ne gère pas la résolution des index? C'est une convention?

Reply:
Without it, it doesn't compile. 
But perhaps we can change this behaviour in the TS configuration (`moduleResolution` ?)

Reply:
Related SO question : https://stackoverflow.com/questions/44028806/should-the-index-ts-be-resolved-by-typescript-as-default-module-file

Reply:
Does this mean TS people consuming the lib need to point `sigma/renderers/webgl/index`?

Reply:
I will say it depends of their TS configuration.
I have a react TS project where it's not needed to put the index. So I think we can find a TS config that works for the omission of 'index'.